src/quantum/config.js

```diff
@@ -51,75 +52,159 @@ const PLATFORMS = [
     bits: 32,
     os: 'win',
     label: 'Firefox',
-    framework: 10,
-    platform: 'windows7-32',
-    option: 'pgo',
-    project: 'mozilla-central',
+    platform: "win32",
+    seriesConfig: {
+      and: [
```

Comment:
This is incredible. Please go back or find another reviewer.

Reply:
I thought you might like this; your original suggestion for this file was to split out the `seriesConfig` from the rest of the properties, and this proves you were right.  But, I also needed a way to declare a number of specific platforms and combinations that this series can be found running on. I can not go back, I need a solution.

Rather than

```
seriesConfig: {
    framework: 10,
    platform: 'windows7-32',
    option: 'pgo',
    project: 'mozilla-central',
}
```

which assumes those properties must match the series, the new format is more explicit: Saying that the properties must be `eq`ual:

```
seriesConfig: { eq: {
    framework: 10,
    platform: 'windows7-32',
    option: 'pgo',
    project: 'mozilla-central',
}}
```

but since there a couple platform combinations, we use some Boolean logic to combine them.

```js
{"seriesConfig":{"and":[
    {"or":[
        {"eq":{"platform":"windows7-32","options":"pgo"}},
        {"eq":{"platform":"windows7-32-shippable","options":"opt"}}
    ]},
    {"eq":{"framework":10,"repo":"mozilla-central"}}
]}}
```

It is unfortunate that the linter spreads this over so many lines.  Here is the Javascript for the same:

```js
  ({ platform, framework, repo, options }) =>
    framework === 10 &&
    repo === 'mozilla-central' &&
    ((platform === 'windows10-64-shippable' && options === 'opt') ||
      (platform === 'windows10-64' && options === 'pgo'))
```

Personally, I find the `{and:[]}` and `{or:[]}` are easier to read than infix Boolean operators with brackets.

I agree, there can be a good argument for avoiding these JSON expressions, and using some specialized format; maybe an array of `seriesConfig` for multiple platforms.  I found that not suitable because it is not general enough: We also have instances of suite names changing (https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/quantum/index.jsx#L257), which multiplied by the number of platforms is [6 signatures](https://treeherder.mozilla.org/perf.html#/graphs?timerange=5184000&series=mozilla-central,1503356,1,1&series=mozilla-central,1521830,1,1&series=mozilla-central,1539226,1,1&series=mozilla-central,1552932,1,1&series=mozilla-central,1902204,1,1&series=mozilla-central,1934089,1,1) (well, it was before the database rewrite).  Plus, there are future expressions; like pulling load time over all the suites so we can calculate aggregates.